Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -245,6 +246,20 @@ private class ReaderModeButton: UIButton {
 }
 
 private class DisplayTextField: UITextField {
+    weak var accessibilityActionsSource: AccessibilityActionsSource?
+
+    override var accessibilityCustomActions: [AnyObject]! {
+        get {
+            if !editing {
+                return accessibilityActionsSource?.accessibilityCustomActionsForView(self)
+            }
+            return super.accessibilityCustomActions
```

Comment:
Good point about the `editing` which is no longer needed. But the getter (and therefore setter) are still needed - the method `accessibilityCustomActionsForView` can return different actions over time - see `BrowserViewController.locationActionsForURLBar`. Basically `DisplayTextField` is constantly not `editing`, so you can think that the `if` branch (with calling the `accessibilityCustomActionsForView`) is always taken.